### DISCUSSION TITLE
PerfGraph

### DISCUSSION THREAD

**Discussion by singhgp4321** [idaholab/moose#16374] [Q&A General] [answered]
I am removing this post since I found the answer.

**Comment by cticenhour** [comment, marked as answer]
In the future - if you find an answer yourself, it would be beneficial to others to comment on what the solution you discovered was and leave your original question as a resource. We want this new Discussions forum to be a searchable database of user issues and support, much like the old mailing list. 

Thank you for submitting a question, and I'm glad you were able to find the answer!
